src/engine/enginemaster.cpp

```diff
@@ -97,6 +100,13 @@ EngineMaster::EngineMaster(UserSettingsPointer pConfig,
 
     m_pMasterDelay = new EngineDelay(group, ConfigKey(group, "delay"));
     m_pHeadDelay = new EngineDelay(group, ConfigKey(group, "headDelay"));
+    m_pInputLatencyCompensationDelay = new EngineDelay(group,
+        ConfigKey(group, "inputLatency"));
+    m_pInputLatencyCompensationHeadphonesDelay = new EngineDelay(group,
+        ConfigKey(group, "headMixDelay"));
```

Comment:
We have two use cases: 
1. Recording: In this case, the user does not hear the master and the talk over channel. It is a good idea not to have any delay in the headphones to keep a low latency setup. 
2: Dancefloor: The user hears the master. In this case the latency compensation does not work. The user will hear a double base. In this case a speaker distance compensation makes sense. This is the opposite delay setup compared to 1.  
1. Can also used if you have a closed booth or closed headphones, to ban the master sound form the users ears.